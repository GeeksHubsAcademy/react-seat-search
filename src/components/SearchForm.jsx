import React from 'react'

const Searchform = () => {
    return (
        <div>

        </div>
    )
}

export default Searchform
