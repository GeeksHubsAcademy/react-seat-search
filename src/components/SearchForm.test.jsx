import SearchForm from './SearchForm';
import { shallow } from 'enzyme';

describe('Search form unit test', () => {
  test('it should render without crashing', () => {
    const wrapper = shallow(<SearchForm />);
    expect(wrapper).toMatchSnapshot();
  });
});
