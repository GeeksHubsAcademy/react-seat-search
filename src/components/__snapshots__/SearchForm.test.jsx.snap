// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search form unit test it should render without crashing 1`] = `ShallowWrapper {}`;
